Sources/Kevin/Core/Kevin.swift

```diff
@@ -13,10 +13,14 @@ final public class Kevin {
     public var theme = KevinTheme()
     public var locale = Locale(identifier: "en") {
         didSet {
-            if !KevinLocaleManager.supportedLocales.contains(locale.identifier) {
+            var identifier = String(locale.identifier.prefix(2))
```

Comment:
What was the root cause of issue here? Is `locale.identifier` longer than 2 letters?

Reply:
In case in iOS settings you have "preferred language" set to Polish, but locale set to Latvia - locale identifier would be "pl_LV" and our check for supported locales will fail and locale will fall back to English ("en"), because `supportedLocales` doesn't account for `_LV` part (and there is no need to either)

Reply:
👍